source/_includes/installation/operating_system.md

```diff
@@ -300,6 +300,10 @@ _All these can be extended if your usage calls for more resources._
   content: |
     <div class='note warning'>
         Hyper-V does not have USB support
+       
+        Hyper-V in Windows on ARM is supported from Home Assistant OS 10.0 generic aarch64. Before you create a virtual machine, an image conversion should be done.
```

Comment:
I think this means it is not supported, as 10.0 doesn't exist as of yet.

Reply:
@frenck It is verified on HAOS 10.0 20221210 dev version. So I use the `next` branch for the upcoming version of Home Assistant. Or maybe this merge should be pending until 10.0 released?

Reply:
> It is verified on HAOS 10.0 20221210 dev version

Sure, but that isn't a public stable release, and, thus should not be listed in our public stable documentation.

../Frenck